### ISSUE TITLE
crate 0.55.2 - es.config is no longer supported

### ISSUE THREAD

**Issue by schroedingerdb** [crate/docker-crate#60]
Only a hint to update the readme, I mean this
https://github.com/crate/docker-crate#use-custom-crate-configuration

Here my example:

```
cat /tmp/conf/crate.yml
 discovery:
  zen.minimum_master_nodes: 2
  
 gateway:
  recover_after_nodes: 3
  recover_after_time: 5m
  expected_nodes: 3
```

```
docker run --name crate-1 -p 4200:4200 -p 4300:4300 -v /tmp/conf:/conf crate:0.55.2 crate -Des.config=/conf/crate.yml -Des.cluster.name=my-cluster
[2016-08-11 12:31:48,737][INFO ][bootstrap                ] es.config is no longer supported. crate.yml must be placed in the config directory and cannot be renamed.
```

I'm really sad about using something like this
```
docker run -d -p 4200:4200 -p 4300:4300 \
    --name crate1-container \
    --volume /mnt/data:/data \
    --ulimit nofile=65535 \
    --ulimit memlock=9223372036854775807 \
    --env CRATE_HEAP_SIZE=8g \
        crate/crate:latest \
        crate \
          -Des.cluster.name=crate-cluster \
          -Des.node.name=crate1 \
          -Des.transport.publish_port=4300 \
          -Des.network.publish_host="$HOST" \
          -Des.multicast.enabled=false \
          -Des.discovery.zen.ping.unicast.hosts="$HOSTS" \
          -Des.discovery.zen.minimum_master_nodes=2
```

**Comment by chaudum** [comment]
hi @schroedingerdb 
thanks for pointing out that the documentation is outdated! we'll take care and provide the correct way how to use a custom configuration.

PS: contributions also to the documentation are very welcome 😎

**Comment by schroedingerdb** [comment]
Solved with 2f1eea6e5714d5919f343c7a37b47a0baf081434
Thanks :D
